app/src/main/res/values/strings.xml

```diff
@@ -361,31 +361,31 @@
     <string name="error_no_activity_to_view">Could not find an app to open the content with</string>
     <string name="manage_clients">Manage Clients</string>
     <string name="apps">Apps</string>
-    <string name="generate">Generate</string>
-    <string name="scan">Scan</string>
+    <string name="generate">Generate…</string>
+    <string name="scan">Scan…</string>
     <string name="using_qr_code">Using QR Code</string>
     <string name="authors">Authors</string>
     <string name="author">Author</string>
-    <string name="error_communication_security_credentials">Credentials mismatch (needs pairing)</string>
+    <string name="error_communication_security_credentials">Mismatching credentials (needs pairing)</string>
     <string name="invalidate">Invalidate</string>
     <string name="deny">Deny</string>
     <string name="error_communication_security">A security error occurred</string>
     <string name="nickname">Nickname</string>
     <string name="root_folder">Root</string>
     <string name="add_storage_access">Add Storage Access…</string>
-    <string name="clear_storage_list">Clear Storage List</string>
+    <string name="clear_storage_list">Clear Storage List…</string>
     <string name="storage_folder">Storage Folder</string>
     <string name="change_storage_folder">Change Storage Folder</string>
-    <string name="storage_folder_notice">The folder to save contents in</string>
+    <string name="storage_folder_notice">The folder content is saved in</string>
```

Comment:
This is not grammatically correct?

Reply:
It is, but I see the confusion. Run on sentence or whatever it is called.

Reply:
The text will be an overline, so it should just describe the text below.

![Screenshot_20210828-171557_TShot_DEBUG](https://user-images.githubusercontent.com/10258162/131220907-77644f7c-4b7f-413d-8ed0-9c15595f3c15.png)

Reply:
```suggestion
    <string name="storage_folder_notice">Save in…</string>
```